build/blueprints/component/files/src/patternfly/components/__moduleName__/docs/code.md

```diff
@@ -6,12 +6,14 @@ For example for buttons: Always add a modifier class to add color to the button.
 
 ## Accessibility
 
-Write accessibility, aria tags, tab index and other notes to make this component accessible.
+| Attribute | Applied | Outcome |
+| -- | -- | -- |
+| `role` or `aria` | `{{bemName}}` |  accecibility notes. |
+
 
 ## Usage
 
-| Class | Usage |
-| -- | -- |
-| `.class-name-here` **Applied to:** `<tags-here>` |  **Outcome:** Outcome-here **Required:** Yes/No **Remarks:** Remarks-here |
-| Example: `.btn` **Applied to:** `<a>`, `<button>` |  **Outcome:** Initiates a button **Required:** Yes **Remarks:** Always use it with a modifier class. |
-| `.btn-secondary` **Applied to:** `.btn` | **Outcome:** Adds secondary color **Required:** Yes |
\ No newline at end of file
+| Class | Applied | Outcomete |
```

Comment:
Should it say "Applied to"?
Also, "Outcome"

Reply:
I found myself wanting to use "Applied To" as well. I think that would go for both tables.